JitsiTrackError.js

```diff
@@ -29,109 +29,111 @@ TRACK_ERROR_TO_MESSAGE_MAP[JitsiTrackErrors.TRACK_MUTE_UNMUTE_IN_PROGRESS]
 TRACK_ERROR_TO_MESSAGE_MAP[JitsiTrackErrors.NO_DATA_FROM_SOURCE]
     = "The track has stopped receiving data from it's source";
 
+
+// FIXME: Using prototype inheritance because otherwise instanceof is not
```

Comment:
The comment is a bit misleading about the bug. From the issues description one can figure out that it's not recommended to extend a built-in type (and so does the PR description), but the comment looks like we're waiting for the bug to be fixed.